pkg/virt-handler/vm.go

```diff
@@ -2640,13 +2640,18 @@ func (d *VirtualMachineController) updateDomainFunc(old, new interface{}) {
 }
 
 func (d *VirtualMachineController) finalizeMigration(vmi *v1.VirtualMachineInstance) error {
+	const errorMessage = "failed to finalize migration"
 	client, err := d.getVerifiedLauncherClient(vmi)
 	if err != nil {
 		return err
 	}
-	err = client.FinalizeVirtualMachineMigration(vmi)
-	if err != nil {
-		log.Log.Object(vmi).Reason(err).Error("failed to finalize migration")
+
+	if err := isolation.AdjustQemuProcessMemoryLimits(d.podIsolationDetector, vmi); err != nil {
+		log.Log.Object(vmi).Reason(err).Warning(errorMessage)
```

Comment:
Don't we want to return here ? (I honestly don't know).

Reply:
Changing QEMU memlock rlimits doesn't affect the migration result, 
there was shot discussion about it  https://github.com/kubevirt/kubevirt/pull/5558#discussion_r625352942

Reply:
Yep, we shouldn't fail the migration because of this.
However, @ormergi perhaps we should emit an event, so it will be visible on the vmi object and not buried in the logs.
More importantly [here](https://github.com/kubevirt/kubevirt/blob/8cfc7b32b62cd451396d47448083029b7313ad71/pkg/virt-launcher/virtwrap/live-migration-target.go#L38), I think

Reply:
No problem I will add an event to `AdjustQemuProcessMemoryLimits` instead of waning log message
I prefer to add event to https://github.com/kubevirt/kubevirt/blob/8cfc7b32b62cd451396d47448083029b7313ad71/pkg/virt-launcher/virtwrap/live-migration-target.go#L38 
on a follow up PR if its ok with you